protocol/contracts/v2/TreasuryReservesVault.sol

```diff
@@ -174,11 +181,20 @@ contract TreasuryReservesVault is ITreasuryReservesVault, TempleElevatedAccess {
         StrategyConfig storage strategyConfig = strategies[strategy];
         strategyConfig.underperformingEquityThreshold = underperformingEquityThreshold;
 
-        address _token;
+        mapping(IERC20 => uint256) storage _tokenCredits = strategyTokenCredits[strategy];
+
+        uint256 _ceiling;
+        IERC20 _token;
         uint256 _length = debtCeiling.length;
         for (uint256 i; i < _length; ++i) {
-            _token = debtCeiling[i].asset;
-            strategyConfig.debtCeiling[IERC20(_token)] = debtCeiling[i].balance;
+            _token = IERC20(debtCeiling[i].asset);
```

Comment:
gas: cache `debtCeiling[i]` to a variable

Reply:
see the later commit - this is already done 👍